docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -65,8 +72,14 @@ image::case_roles.png[Case Roles]
 * `manager` - the employee's manager; the person who will approve or deny the requested hardware. The role cardinality is set to `1`, which means that only one person or group can be assigned to this role.
 * `supplier` - the available suppliers of IT hardware in the system. There is usually more than one supplier.
```

Comment:
These bulleted descriptions should use a colon followed by capital, "owner: The employee ...". This is consistent with the rest of your doc and with our general style throughout the docs.

Also in this chapter but not in this PR: "process level and case level variables" > "process-level and case-level variables (again, technically, but your call)